src/main/java/core/basesyntax/service/Return.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitBox;
+import java.util.Map;
+
+public class Return extends Operation {
```

Comment:
I think it would be easier to create the interface, not an abstract class. But it is up to you.

Reply:
I used it only because of constructors